pallets/compounding-interest/Cargo.toml

```diff
@@ -1,38 +1,29 @@
 [package]
 name = "compounding-interest"
-version = "2.0.0"
+version = "3.0.0"
 edition = "2018"
 authors = ['Substrate DevHub <https://github.com/substrate-developer-hub>']
 repository = 'https://github.com/substrate-developer-hub/recipes'
 description = "A savings account that uses fixed point arithmetic to implement compounding interest"
 license = "GPL-3.0-or-later"
 
-[package.metadata.substrate]
-categories = [
-	"pallet",
-	"tokens",
-	"accounts",
-	"tutorial",
-	"recipe",
-]
-compatibility_version = "2.0.0"
-
 [dependencies]
-substrate-fixed = { git = 'https://github.com/encointer/substrate-fixed.git', tag = "v0.5.4+sub_v0.1" }
-parity-scale-codec = { version = "1.3.0", features = ["derive"], default-features = false }
+substrate-fixed = { git = 'https://github.com/encointer/substrate-fixed.git', rev = "b33d186888c60f38adafcfc0ec3a21aab263aef1" }
```

Comment:
@JoshOrndorff is this the best way here, pinning substrate-fixed to a specific commit?

Reply:
The recipes now uses `parity-scale-codec = "2.0.0"`. In order t obe compatible, al dependencies must also use something from the `2.x` series. `substrate-fixed` only made this update in the most recent commit (the one I pinned). So for now, that is the only way.

If @brenzi makes a new tag, or publishes to crates.io we could use one of those instead.

Reply:
Or we could point to `branch = "master"` and rely on the cargo.lock to pin the commit. That's probably best actually.